conn.go

```diff
@@ -23,6 +24,7 @@ import (
 	"unicode"
 
 	"github.com/lib/pq/oid"
+	"github.com/spaolacci/murmur3"
```

Comment:
I don't think we want to introduce an external dependency here, since we have none right now. Is there a way to avoid that? But johto's comments are probably more germane.